docs/source/dev_guide/algorithms/dbscan/computation-batch.rst

```diff
@@ -91,6 +94,10 @@ For more details, see :ref:`algorithms`.
          
          By default all weights are equal to :math:`1`.
 
+       .. note::
+
+         This parameter is not supported on GPU.
```

Comment:
What it means for user? To be ignored? Exception to be raised?

Reply:
@amgrigoriev could you clarify?

Reply:
Weights are ignored

Reply:
fixed